Yoast/Tests/WP/RequiredParametersUnitTest.inc

```diff
@@ -0,0 +1,40 @@
+<?php
+
+// Missing autoload option.
+update_option( 'a', 'b' );
+
+// Ok.
+update_option( 'a', 'b', true );
+
+// Ok.
+update_option( 'a', 'b', false );
+
+// Ok.
+update_option( 'a', 'b', 'yes' );
+
+// Ok.
+update_option( 'a', 'b', 'no' );
+
+// Ok.
+update_option( 'a', 'b', null );
+
+// Missing autoload option.
+add_option( 'a', 'b' );
+
+// Missing autoload option.
+add_option( 'a', 'b', '' );
+
+// Ok.
+add_option( 'a', 'b', '', 'yes' );
+
+// Ok.
+add_option( 'a', 'b', '', 'no' );
+
+// Ok.
+add_option( 'a', 'b', '', null );
```

Comment:
`null` is not a valid value to pass to `add_option()`. See my remark in my overall review comment.